website/docs/whats-new/release-notes/partials/design-tokens.md

```diff
@@ -0,0 +1,82 @@
+ **`npm package: @hashicorp/design-system-tokens`**
+
+## 1.8.0
+
+**Month Day, Year**
+
+- `Design tokens` added color tokens for “Vault Secrets” product. 
+
+- `IconTile` updated component to include `vault-secrets` product option.
+
+[(#1640)](https://github.com/hashicorp/design-system/pull/1640) Thanks [@didoo](https://github.com/didoo)!
+
+## 1.7.0
+
+**Month Day, Year**
+
+- `marketing` added JSON output format for target (and in the process refactored internal logic for tokens generation). [(#1513)](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)! 
+
+## 1.6.0
+
+**Month Day, Year**
+
+- Add design token for loading state icon on search input. [(#1452)](https://github.com/hashicorp/design-system/pull/1452) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+## 1.5.0
+
+**Month Day, Year**
+
+- `TooltipButton` added new component and `hds-tooltip` modifier.
+
+- `Tooltip` added design tokens.
+
+- [(#1283)](https://github.com/hashicorp/design-system/pull/1283) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
+
+## 1.4.2
+
+**Month Day, Year**
+
+- `SideNav` added design tokens [(#1313)](https://github.com/hashicorp/design-system/pull/1313) Thanks [@didoo](https://github.com/didoo)! 
+
+## 1.4.1
+
+**Month Day, Year**
+
+- Scope group layout styles to nested child components.
+
+- Fix bug with `--token-pagination-child-spacing-vertical` value so that it adds "px" unit.
+
+- [(#1204)](https://github.com/hashicorp/design-system/pull/1204) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
```

Comment:
I'm noticing that component and token changes in the same PR are turning out to be less of an edge case. Cleaning this up to include only the token changes and updating the formatting to add the PR link and contribution acknowledgment inline.

```suggestion
- Fixed a bug with `--token-pagination-child-spacing-vertical` value so that it adds "px" unit. [#1204](https://github.com/hashicorp/design-system/pull/1204) Thanks [@KristinLBradley](https://github.com/KristinLBradley)!
```